app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -262,7 +261,6 @@ class AttendeeViewModel(
                     mutableProgress.value = false
                 }.subscribe({
                     orderIdentifier = it.identifier.toString()
-                    mutableMessage.value = "Order created successfully!"
```

Comment:
Why?

Reply:
This shows up twice because it's the function it calls also has same sanckbar.

Reply:
This is okay right?

Reply:
Changed it slightly because, in case of free tickets, confirmorderstatus is called which gives order created successfully snackbar. So changed it so that snackbar is shown when paid ticket because paid ticket has no such snackbar. Removes repetition of snackbar "Order created successfully!" in case of free tickets.

Reply:
We want to show snackbar in both free and paid cases

Reply:
It is shown in both cases. In free it was showing twice.

Reply:
In case of free, confirmorderstatus is being called which shows same snackbar